pkg/apis/integreatly/v1alpha1/apicuriodeployment_types.go

```diff
@@ -13,18 +13,19 @@ const (
 )
 
 type ApicurioDeploymentSpec struct {
-	Version         string     `json:"version"`
-	AppDomain       string     `json:"app_domain"`
-	Template        string     `json:"template"`
-	ExternalAuthUrl string     `json:"external_auth_url"`
-	AuthRealm       string     `json:"auth_realm"`
-	JvmHeap         [2]string  `json:"jvm_heap"`
-	MemLimit        [2]string  `json:"mem_limit"`
+	Version         string    `json:"version"`
```

Comment:
I think we should remove version from here. The operator should decide the version not the user